var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -52,7 +52,7 @@ class Hdf5(Package):
     variant('cxx', default=True, description='Enable C++ support')
     variant('fortran', default=True, description='Enable Fortran support')
 
-    variant('mpi', default=False, description='Enable MPI support')
+    variant('mpi', default=True, description='Enable MPI support')
```

Comment:
For comparison, the following packages default to `+mpi`:

- bertini, cp2k, dakota, dealii, elk, espresso, fenics, gromacs, hmmer, hoomd-blue, intel-parallel-studio, kripke, llvm, meep, mumps, netcdf, paradiseo, petsc, plumed, py-meep, sundials, swiftsim, tau, valgrind

and the following packages default to `~mpi`:

- arpack-ng, caliper, fftw, gmsh, hdf5, hpx5, mfem, paraview, pgi, py-h5py, scotch, turbomole, zoltan